contentful/apps
# Feature INTEG-549: Content editors can see the ecommerce product linked to a given content entry

## Purpose

As part of the e-comm stack, Content Editors will need to see ecommerce product data in a field attached to an entry. 

## Approach

Building off what was done in: https://github.com/contentful/apps/pull/3498

## Testing steps

- Configure Shopify app in your space with our partner credentials (See Shopify (Production) space in Apps Development org), hit Save
- Navigate to an entry, click "Add Product" (will randomize between a real product id or a fake one)
- Validate that product details show up, or Not Found is shown for missing product
